ipaserver/plugins/groupfas.py

```diff
@@ -173,16 +176,13 @@ def group_find_fas_postcb(self, ldap, entries, truncated, *args, **options):
 def group_mod_fas_precb(self, ldap, dn, entry, *keys, **options):
     """Add fasgroup object class and related attributes.
     """
-    fas_params = [
-        option for option in options
-        if option.startswith("fas") and option != "fasgroup"
-        # fasgroup is a flag, it's always present: filter it out.
-    ]
-    if options.get("fasgroup", False) or fas_params:
-        entry_oc = ldap.get_entry(dn, ["objectclass"])["objectclass"]
-        if not self.obj.has_objectclass(entry_oc, "fasgroup"):
+    if any(option.startswith("fas") for option in options):
```

Comment:
Same remark here about flags always being in the options dict and the filtering I did above.